### PR TITLE
Cavy/add focus types

### PR DESCRIPTION
**Pull request by austinpgraham**
Please fill in this template.

- [ x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x] Test the change in your own code. (Compile and run.)
- [ x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/pixielabs/cavy/pull/160/files
- [x ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by austinpgraham** [comment]
@AbigailMcP Type definition PR!

File: types/cavy/index.d.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ export class TestScope {
     beforeEach(fn: () => void): void;
     press(identifier: string): Promise<void>;
     fillIn(identifier: string, str: string): Promise<void>;
+    focus(identifier: string): Promise<void>;
```
Comment (peterblazejewicz) [inline types/cavy/index.d.ts]: As this is a new api, can we amend the inline doc, like that one from the upstream change?
```text
Returns a promise, use await when calling this function.
Promise will berejected if the component is not found.
```

**Comment by peterblazejewicz** [comment]
Note for mantainer, this change is not yet relased, would this require version change?
https://github.com/pixielabs/cavy/pull/160/files

**Comment by AbigailMcP** [comment]
> Note for mantainer, this change is not yet relased, would this require version change?
> https://github.com/pixielabs/cavy/pull/160/files

This change is now released, and is a new feature i.e. minor bump to v3.2.0.

**Review by AbigailMcP — CHANGES_REQUESTED** [review]
@austinpgraham as per @peterblazejewicz comment, the Cavy version needs to be changed to cavy 3.2 at the top of `types/cavy/index.d.ts` to match the current release. Thanks!

**Comment by austinpgraham** [comment]
@peterblazejewicz @AbigailMcP Done!

**Review by peterblazejewicz — APPROVED** [review]
LGTM
It's already released in 3.2:
https://cavy.app/docs/api/helpers#focusidentifier

**Review by AbigailMcP — APPROVED** [review]

**Pull request merged**